tests/testthat/test-Model-class.R

```diff
@@ -68,6 +78,36 @@ test_that("MCMC computes correct values for LogisticNormal model", {
 
 ## dose ----
 
+test_that("dose computes correct values for LogisticNormal model", {
+  data <- h_get_data()
+  model <- h_get_logistic_normal()
+  options <- h_get_mcmc_options(small = TRUE, fixed = TRUE)
+
+  samples <- mcmc(data = data, model = model, options = options)
```

Comment:
I think in this case we don't need real MCMC samples. Instead we are interested if the dose function works correctly and that we can almost read that manually from the test code here.
so I would recommend to construct the samples manually with like 3 different values in it and then create explicit expectation from that and compare.
```suggestion
samples <- Samples(
  data = list(
    alpha0 = c(0, -1, 1),
    alpha1 = c(0, 1, -1)
  ),
  options = McmcOptions(burnin = 0L, step = 1L, samples = 3L)
)
```